packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -270,16 +270,32 @@ class SkiaGoldClient {
         for(Map<String, dynamic> ignore in ignores) {
           final List<String> ignoredQueries = ignore['query'].split('&');
           final String ignoredPullRequest = ignore['note'].split('/').last;
+          final DateTime expiration = DateTime.parse(ignore['expires']);
           if (ignoredQueries.contains('name=$testName') &&
-            ignoredPullRequest == pullRequest) {
+            expiration.isAfter(DateTime.now())) {
```

Comment:
Once this expires and the image still hasn't been triaged: We should inform users here why their test may be failing and how to resolve it (triage the image).